sdk/storage/storage-blob/src/pollers/BlobStartCopyFromUrlPoller.ts

```diff
@@ -59,7 +60,9 @@ export interface BlobBeginCopyFromUrlPollState
  * @ignore
  */
 export interface BlobBeginCopyFromURLPollOperation
-  extends PollOperation<BlobBeginCopyFromUrlPollState, BlobBeginCopyFromURLResponse> {}
+  extends PollOperation<BlobBeginCopyFromUrlPollState, BlobBeginCopyFromURLResponse> {
+    lastResponse: unknown;
```

Comment:
instead of `unknown` I could use `BlobStartCopyFromURLResponse | BlobAbortCopyFromURLResponse | BlobGetPropertiesResponse | undefined `

Reply:
So far only Storage is asking for this support, if we want to move this to lro then `unknown` probably.